packages/contracts-bedrock/scripts/upgrades/PostSherlock.s.sol

```diff
@@ -110,6 +110,16 @@ contract PostSherlockL1 is SafeBuilder {
         require(PROXY_ADMIN.getProxyImplementation(prox.L1ERC721Bridge).codehash == impl.L1ERC721Bridge.codehash, "L1ERC721Bridge codehash");
     }
 
+    /**
```

Comment:
I prefer to leave this `run` in the `SafeBuilder` for now and instead wrap it with overloaded `run` if we need more args. Then in the future we can rearchitect a bit more easily

Reply:
Note this will require then to run the `DeleteOutput` script with `--sig` _or_ to hardcode the index to delete in the script. No way to get around this unless the `uint256 index` is an argument in `run()`.